frontend/components/DashboardItem.vue

```diff
@@ -0,0 +1,136 @@
+                        <span class="text-text dark:text-white">
+                            <T :keyName="roleType" />
+                        </span>
+                    </h3>
+                </NuxtLink>
+            </div>
+        </div>
+        <div id="journey-mobile"
+            class="lg:hidden bg-card dark:bg-card-dark border border-border rounded-md p-1 md:p-2 min-w-36 h-32">
+            <div class="flex justify-between">
+                <NuxtLink :to="link" class="overflow-hidden whitespace-nowrap overflow-ellipsis">
+                    <h1 class="font-semibold text-xl overflow-hidden whitespace-nowrap overflow-ellipsis"
+                        v-tooltip.top="name">
+                        {{
+                name }}</h1>
+                </NuxtLink>
+                <Button type="button" icon="pi pi-ellipsis-v" @click="toggle" aria-haspopup="true"
+                    aria-controls="overlay_menu" class="justify-end" />
+            </div>
+            <NuxtLink :to="link">
+                <h2 class="text-lg font-medium -mt-1.5 overflow-ellipsis overflow-hidden whitespace-nowrap"
+                    v-tooltip.bottom="destination">{{ destination }}</h2>
+                <h3
+                    class="border-b-2 border-dashed border-border-grey dark:border-input-placeholder text-xs md:text-sm mt-1.5">
+                    <span class="text-border mr-0.5">
+                        <T keyName="dashboard.date" />
+                    </span>
+                    <br class="sm:hidden">
+                    <span class="text-text dark:text-white whitespace-nowrap">{{ format(from, "dd/MM/yyyy") }} - {{
+                format(to,
+                    "dd/MM/yyyy") }}</span>
+                </h3>
+                <h3
+                    class="mt-1 border-b-2 border-dashed border-border-grey dark:border-input-placeholder text-xs md:text-sm">
+                    <span class="text-border mr-0.5">
+                        <T keyName="dashboard.role" />
+                    </span>
+                    <span class="text-text dark:text-white">
+                        <T :keyName="roleType" />
+                    </span>
+                </h3>
+            </NuxtLink>
+        </div>
+        <Menu ref="menu" id="overlay_menu" :model="props.role === 1 ? itemsJourneyGuide : itemsJourneyMember"
+            class="bg-input dark:bg-input-dark" :popup="true"
+            :pt="{ root: { class: 'font-nunito bg-input dark:bg-input-dark' }, menuitem: { class: 'bg-input dark:bg-input-dark hover:bg-cta-bg-light dark:hover:bg-cta-bg-dark rounded-md text-text dark:text-white' }, content: { class: 'bg-input dark:bg-input-dark hover:bg-cta-bg-light dark:hover:bg-cta-bg-dark rounded-md text-text dark:text-white' }, submenuHeader: { class: 'text-input-placeholder dark:text-text-light-dark bg-input dark:bg-input-dark' }, label: { class: 'text-text dark:text-white' }, icon: { class: 'text-text dark:text-white' } }" />
+    </div>
+</template>
```

Comment:
missing empty line at end of file, can we maybe add a rule to our auto-formatting to fix this?

Reply:
"files.insertFinalNewline": true in settings.json